extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -57,6 +63,32 @@ class AmpStickyAd extends AMP.BaseElement {
   buildCallback() {
     this.viewport_ = this.getViewport();
     this.element.classList.add('i-amphtml-sticky-ad-layout');
+
+    // Setting padding-bottom to avoid iPhone home bar
+    if (isExperimentOn(this.win, 'sticky-ad-padding-bottom')) {
+      const experimentInfoMap = /** @type {!Object<string,
+        !../../../src/experiments.ExperimentInfo>} */ ({
+        [[STICKY_AD_PADDING_BOTTOM_EXP.id]]: {
+          isTrafficEligible: () => true,
+          branches: [
+            [STICKY_AD_PADDING_BOTTOM_EXP.control],
+            [STICKY_AD_PADDING_BOTTOM_EXP.experiment],
+          ],
+        },
+      });
+      randomlySelectUnsetExperiments(this.win, experimentInfoMap);
+      if (
+        getExperimentBranch(this.win, STICKY_AD_PADDING_BOTTOM_EXP.id) ==
+        STICKY_AD_PADDING_BOTTOM_EXP.experiment
+      ) {
+        setStyle(
```

Comment:
nit: will we be doing this in css if we launch? If so maybe move to CSS now and add the class here?

Reply:
Since we do not need to gate on device, my plan is to move this style directly into the `amp-sticky-ad` selector. Therefore for this experiment, it seems easy just to `setStyle`.

Reply:
sgtm